lib/chef/resource/execute.rb

```diff
@@ -135,12 +135,18 @@ def timeout(arg = nil)
         )
       end
 
-      def user(arg = nil)
-        set_or_return(
-          :user,
-          arg,
-          :kind_of => [ String, Integer ]
-        )
+      property :user, [ String, Integer ]
+
+      property :domain, String
+
+      property :password, String, sensitive: true
+
+      def sensitive(args = nil)
```

Comment:
`property :password, String, sensitive: true` was not setting `sensitive=true` for `Chef::Resource::Execute` even if password is specified. Hence I had to define `sensitive` again.